File: backend/servicios-usuario/src/main/java/com/phonecinema/serviciosusuario/dto/AuthResponseDTO.java
package com.phonecinema.serviciosusuario.dto;

import lombok.Data;

@Data
public class AuthResponseDTO {
    private Long id;
    private String nombre;
    private String email;
    private String rol;
    private String fotoPerfilUrl;
    private String token;

    public AuthResponseDTO(Long id, String nombre, String email, String rol, String fotoPerfilUrl, String token) {
        this.id = id;
        this.nombre = nombre;
        this.email = email;
        this.rol = rol;
        this.fotoPerfilUrl = fotoPerfilUrl;
        this.token = token;
    }
}

File: frontend/frontend/app/src/main/java/com/example/phonecinemaapp/data/remote/dto/AuthResponseDto.kt
import com.google.gson.annotations.SerializedName

data class AuthResponseDto(
    @SerializedName("idUsuario") val id: Long,
    val nombre: String,
    val email: String,
    val fotoPerfilUrl: String?,
    val rol: String,
    val mensaje: String,
    val token: String?
)